nastaveni/nastaveni.php

```diff
@@ -175,7 +176,8 @@
 //ceny věcí
 $GLOBALS['KURZ_EURO']=KURZ_EURO;
 $GLOBALS['UCET_CZ']=UCET_CZ;
-$GLOBALS['UCET_SK']=UCET_SK;
+$GLOBALS['IBAN']=IBAN;
```

Comment:
Mne to trochu zmátlo, proč tam ta definice globální proměnné je (a i teď mi chvíli trvalo, k čemu a jak by se to používalo - ještě se o tom zmíním osobně). Každopádně imho jsem to pobral a odmazal (ani kurz_euro se nikde nepoužíval jako proměnná)